<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderToFileDialog</class>
 <widget class="QDialog" name="RenderToFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Resolution</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Width:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="txtWidth">
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>1920</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Height:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="txtHeight">
              <property name="minimum">
               <number>56</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>1080</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxKeepAspect">
            <property name="text">
             <string>Keep aspect ratio</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>File</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Name of the file:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="txtFilename">
              <property name="text">
               <string>render.jpg</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnBrowse">
              <property name="text">
               <string>Select file...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grpVideoOptions">
         <property name="title">
          <string>Video options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Framerate:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="txtFramerate">
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>120.000000000000000</double>
              </property>
              <property name="value">
               <double>30.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>ffmpeg:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtFfmpeg"/>
            </item>
            <item>
             <widget class="QPushButton" name="btnFindFfmpeg">
              <property name="text">
               <string>Find ffmpeg</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>verticalSpacer</zorder>
  <zorder>verticalSpacer_2</zorder>
  <zorder>grpVideoOptions</zorder>
  <zorder>label_3</zorder>
  <zorder>grpVideoOptions</zorder>
  <zorder>cbxKeepAspect</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RenderToFileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RenderToFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
